index.html

```diff
@@ -0,0 +1,463 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i
+          class="fas fa-bars hamburger-open-menu"
+          id="hamburger-close-menu"
+        ></i>
+      </button>
+      <nav id="desktop-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+
+      <div class="mobile-menu-container">
+        <i
+          class="fas fa-times fa-2x hamburger-close-menu"
+          id="mobile-close-menu"
+        ></i>
+        <ul>
+          <li class="mobile-nav-link"><a href="#about-me">About</a></li>
+          <li class="mobile-nav-link"><a href="#skillset">Skill</a></li>
+          <li class="mobile-nav-link"><a href="#projects">Portfolio</a></li>
+          <li class="mobile-nav-link"><a href="#contact">Contact</a></li>
+        </ul>
```

Comment:
- [ ] The hamburger menu is still not following the design [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) as pointed out by the previous reviewer. Sticking to the mockups is a good professional practice for every frontend software developer. Please, make sure you implement the changes as pointed out. 
## Current menu as it is :point_down: 
![image](https://user-images.githubusercontent.com/45668340/143851061-5de5b45c-e20f-4fdd-8b6b-4c680e0698f5.png)
## Design template :point_down: 
![image](https://user-images.githubusercontent.com/45668340/143851222-8eedf509-d7c1-4f84-af84-3915a4a8cddc.png)